core/lib/dal/migrations/20241209150000_create_via_votes.up.sql

```diff
@@ -2,6 +2,8 @@ CREATE TABLE IF NOT EXISTS via_votable_transactions (
     l1_batch_number BIGINT NOT NULL REFERENCES l1_batches (number) ON DELETE CASCADE,
     tx_id BYTEA,
     is_finalized BOOLEAN NOT NULL DEFAULT FALSE,
+    is_verified BOOLEAN NOT NULL DEFAULT FALSE,
```

Comment:
Can you explain the purpose of `is_verified` and `l1_batch_status`?

Reply:
The `is_verified` is updated by the verifier after the ZKP is done and the `l1_batch_status` is the output of the verification valid or not valid.

Reply:
I don't think i fully get the idea here. `via_votable_transactions` table was previously used for the voting mechanism (meaning `btc_watch` Sequencer component) and here it looks that we are using the same table for the Verifier db to store the verification status for this one Verifier.
Can you elaborate on this?

Reply:
The `via_votable_transactions`  stores the l1_block data that is ready to be verified. 
### Flow

1. The `btc_watch` detects that there is a new `proof inscription`, it inserts a new row in the `via_votable_transactions`.
2. The verifier task detects that there is a new votable transaction that was not verrified (`is_verified=FALSE`).
3. The verifier starts the ZKP verification. After finishing it update the `via_votable_transactions` columns `l1_batch_status` (if valid proof set to `TRUE` else `FALSE`) and `is_verified=TRUE`.
4. The btc_sender detects that the verification is finished and it needs to create a voting inscription.
5. The voting inscription is created and sent.
6. The l1_batch is finailized

The `via_votable_transactions` should be migrated to the verifer DAL in another PR as it's related to verification.